### PR TITLE
fix: improve callout block settings

### PR DESCRIPTION
**Pull request by mike-schmid**
Ticket: https://app.clickup.com/t/2ry3j8m

### CODE REVIEW

File: packages/callout-block/src/settings.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export const settings: BlockSettings = {
         {
             id: 'type',
             type: 'dropdown',
-            defaultValue: Type.Warning,
+            defaultValue: Type.Info,
```
Comment (Walkeezy) [inline packages/callout-block/src/settings.ts:27]: I would also move `Info` to the top of the dropdown (also like this in the design).

**Review by triggertoo — APPROVED** [review]
looks good 👍

File: packages/callout-block/src/settings.ts (hunk 1 above)
Comment (mike-schmid) [reply packages/callout-block/src/settings.ts:27]: Good point. I've updated the order. Can you please approve again?
thanks

**Comment by mike-schmid** [comment]
close this PR and make a new one because of unverified commits.
new PR: https://github.com/Frontify/guideline-blocks/pull/382

**Pull request closed without merging**